### PR TITLE
Update project metadata

### PR DESCRIPTION
**Pull request by aerlaut**
# Background
#### Link to issue 

https://biomage.atlassian.net/browse/BIOMAGE-964

#### Link to staging deployment URL 

https://ui-agi-metadata.scp-staging.biomage.net/data-management

#### Anything else the reviewers should know about the changes here

# Changes
### Code changes

# Definition of DONE
Your changes will be ready for merging after each of the steps below have been completed:

### Testing
- [X] Unit tests written
- [X] Tested locally with Inframock (with latest production data downloaded with biomage experiment pull)
- [x] Deployed to staging

To set up easy local testing with inframock, follow the instructions here: https://github.com/biomage-ltd/inframock
To deploy to the staging environment, follow the instructions here: https://github.com/biomage-ltd/biomage-utils

### Documentation updates
Is all relevant documentation updated to reflect the proposed changes in this PR?

- [ ] Relevant Github READMEs updated
- [ ] Relevant wiki pages created/updated

### Approvers
- [ ] Approved by a member of the core engineering team
- [ ] (UX changes) Approved by vickymorrison (this is her username, tag her if you need approval)

### Just before merging:
- [ ] After the PR is approved, the `unstage` script in here: https://github.com/biomage-ltd/biomage-utils is executed. This script cleans up your deployment to staging

### Optional
- [ ] Photo of a cute animal attached to this PR

### CODE REVIEW

File: src/components/data-management/ProjectDetails.jsx [hunk 1]
```diff
@@ -468,7 +468,10 @@ const ProjectDetails = ({ width, height }) => {
         record,
         'species',
         rowIdx,
-        (newValue) => dispatch(updateSample(record.uuid, { species: newValue })),
+        (newValue) => {
+          console.log(`${record.uuid} - ${newValue}`);
```
Comment (xverges) [inline src/components/data-management/ProjectDetails.jsx]: Remove?

File: src/pages/data-management/index.jsx [hunk 2]
```diff
@@ -93,10 +93,10 @@ const DataManagementPage = ({ route }) => {
         route={route}
         title='Data Management'
       />
-      <LoadingModal
+      {/* <LoadingModal
```
Comment (xverges) [inline src/pages/data-management/index.jsx]: ?

File: src/components/data-management/ProjectDetails.jsx (hunk 1 above)
Comment (aerlaut) [reply src/components/data-management/ProjectDetails.jsx]: Removed

File: src/pages/data-management/index.jsx (hunk 2 above)
Comment (aerlaut) [reply src/pages/data-management/index.jsx]: Ah yea, I forgot to uncomment them

**Review by xverges — COMMENTED** [review]
lgtm, but would rather have some new unit tests

**Review by xverges — APPROVED** [review]

**Pull request merged**